enhancements/two-nodes-openshift/2no.md

```diff
@@ -178,29 +210,37 @@ This will be achieved by having the same entity that drives the configuration of
 To enable this flow, we propose the addition of a `managedEtcdKind` field which defaults to `Cluster` but will be set to `External` during installation, and will only be respected if the `Infrastructure` CR's `TopologyMode` is `DualReplicaTopologyMode`.
 This will allow the use of a credential scoped to `ConfigMap`s in the `openshift-etcd-operator` namespace, to make the change.
```

Comment:
I think the pacemaker initialization script is still what sets this `managedEtcdKind` to external. If we kick off that script from CEO, we could rework it so that CEO does this step, but IIUC the current intent is for pacemaker to kick off this change.